systemtests/src/main/java/io/enmasse/systemtest/selenium/page/OpenshiftWebPage.java

```diff
@@ -174,6 +174,7 @@ public void openOpenshiftPage() throws Exception {
         if (!waitUntilConsolePage()) {
             throw new IllegalStateException("Openshift console not loaded");
         }
+        checkReachableWebPage();
```

Comment:
In this case you can probably use content of waitUntilConsolePage (3 rows above), because it seems to me like this method was implemented for the same purpose (somewhen in past). But, if new check from checkReachableWebPage is for some reason better, then condition from row 174 can be removed. So this is up to you :)

Reply:
line 177 removed, I think original check is better